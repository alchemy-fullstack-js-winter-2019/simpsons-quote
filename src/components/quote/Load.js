import React from 'react';
import PropTypes from 'prop-types';

const Load = ({ fetch }) => {
  return (
    <div>
      <button onClick={fetch}>Refresh</button>
    </div>
  );
};

Load.propTypes = {
  fetch: PropTypes.func.isRequired
};

export default Load;
