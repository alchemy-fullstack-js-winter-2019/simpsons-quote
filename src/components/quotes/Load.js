import React from 'react';
import PropTypes from 'prop-types';



const Load = ({ fetch }) => {
  return (
    <button onClick={fetch}>Next Quote</button>
  );
};

Load.propTypes = {
  fetch: PropTypes.func.isRequired
};
  
export default Load;
