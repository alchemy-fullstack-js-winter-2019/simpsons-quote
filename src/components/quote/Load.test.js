import React from 'react';
import renderer from 'react-test-renderer';
import Load from './Load';

describe('Load', () => {
  it('matches a snapshot', () => {
    const tree = renderer.create(
      <Load />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
