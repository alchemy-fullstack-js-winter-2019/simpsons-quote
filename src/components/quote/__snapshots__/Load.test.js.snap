// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Load matches a snapshot 1`] = `
<div>
  <button
    onClick={[MockFunction]}
  >
    Refresh
  </button>
</div>
`;
